dmci/api/app.py

```diff
@@ -77,8 +77,11 @@ def post_update():
         @self.route("/v1/delete/<metadata_id>", methods=["POST"])
         def post_delete(metadata_id=None):
             """Process delete command."""
-            md_uuid, md_namespace, err = self._check_namespace_UUID(
-                metadata_id)
+            md_namespace, md_uuid, err = self._check_metadata_id(metadata_id,
+                                                                 self._conf.env_string)
+
+            if err is not None:
+                logger.error(err)
```

Comment:
err here is for api-user. logger is output to dmci internally

Reply:
yes - but this is ok, right?

Reply:
Aren't we already logging something inside of the function? but sure.

Reply:
I removed the logging there to avoid repetition